### ISSUE TITLE
Quick Quote Bibliographical Reference JS

### ISSUE THREAD

**Issue by gasyoun** [sanskrit-lexicon/COLOGNE#341]
If I wan to quote Cologne, I have to do at least 3 clicks (click where from, drag, until where)

Let's take `agni` from MW in https://sanskrit-lexicon.uni-koeln.de/simple/
```
(H1) [Printed book page 5,1]agni m. (√ag, Uṇ. ) fire, sacrificial fire (of three kinds, Gārhapatya, Āhavanīya, and Dakṣiṇa) [ID=890]
--
the number three, Sūryas.  [ID=891]
the god of fire, the fire of the stomach, digestive faculty, gastric fluid [ID=892]
bile, L.  [ID=893]
gold, L.  [ID=894]
N. of various plants Semecarpus Anacardium, Suśr. , Plumbago Zeylanica and Rosea, Citrus Acida [ID=895]
mystical substitute for the letter r [ID=896]
in the Kātantra grammar N. of noun-stems ending in i and u [ID=897]
[Printed book page 1309,1](also) = next, ĀpŚr.  [ID=897.05]
[Printed book page 5,1]agni [cf. Lat. igni-s; Lith. ugni-s; Slav. ognj]. [ID=897.1]
```

**Comment by gasyoun** [comment]
@funderburkjim would you mind if I ask to add such a functionality to @alex-bukach ?

See the `Copy button` code
```
        <div class="input-group-append" aria-live="assertive">
            <button class="btn copy-button d-flex align-items-center" data-copy-from="checksum" type="button"><span class="ms-Icon ms-Icon-action ms-Icon--Copy"></span><span>Copy</span></button>
        </div>
```

https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-5.0.5-windows-x64-installer

![copy](https://user-images.githubusercontent.com/80761/114286433-5f882900-9a67-11eb-9811-c980ff8d1f39.png)

**Comment by funderburkjim** [comment]
I see no reason not to try it.

Looks like you want to add a button and some Javascript to simple-search.

Is there a url where the click to quote functionality can be seen in action?

What is the dot-net reference about?

**Comment by funderburkjim** [comment]
https://github.com/sanskrit-lexicon/csl-apidev/tree/master/simple-search/v1.1a

has the latest non-production code.

We can experiment with it.  

https://sanskrit-lexicon.uni-koeln.de/simple1.1a/

accesses this experimental code.

**Comment by funderburkjim** [comment]
The entry point of the code is list-0.2s_rw.php,
and this is where the UI mostly lives.

**Comment by gasyoun** [comment]
> I see no reason not to try it.

Great.

> Is there a url where the click to quote functionality can be seen in action?

Yes, the copy button at https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-5.0.5-windows-x64-installer
Never seen inside a dictionary.

> What is the dot-net reference about?

Only a copy button UI sample.

**Comment by funderburkjim** [comment]
> copy button UI sample

Is this Github example a similar idea?

![image](https://user-images.githubusercontent.com/6393033/114315574-998c1600-9acd-11eb-96e5-d1671647df4e.png)

**Comment by gasyoun** [comment]
> Is this Github example a similar idea?

Yes, it's identical. And I want such functionality at Cologne. For every line and all text as once. Two modes.
